group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -629,14 +641,10 @@ public void testJoiningExistingNotReadyStreamsGroupMissingTopics() {
                 )
         );
         StreamsGroupHeartbeatRequestData heartbeatToCreateGroup =
-            buildFirstStreamsGroupHeartbeatRequest(groupId, topology, processId, rebalanceTimeoutMs);
+            buildFirstStreamsGroupHeartbeatRequest(groupId, topology, processId, rebalanceTimeoutMs, memberId);
         prepareStreamsGroupAssignment(assignor, heartbeatToCreateGroup.memberId(), "subtopology-id");
-        context.streamsGroupHeartbeat(heartbeatToCreateGroup);
```

Comment:
Removed this initial heartbeat request as the first one contains the member id making assignment and group creation possible with the first heartbeat request.